### PR TITLE
IT-34972 / user, pwd and repoName from JSON file

### PR DESCRIPTION
**Pull request by apgsga-jhe**
Reading info from repoNames.json.
I'll also integrate the modifications suggested by @chhex within #7 ..... but this can already been reviewd.

One question ... : I find repoNames.json not a very good name anymore ... What do you think? What about "apgGradlePluginConfig.json" ?

### CODE REVIEW

**Comment by apgsga-jhe** [comment]
@apgsga-stb , @chhex  : this time it's the correct one :) :)

File: common-repo/src/main/java/com/apgsga/gradle/repo/extensions/RemoteRepo.java [hunk 1]
```diff
@@ -15,31 +15,65 @@
...
+
+	private static final String REPOS_KEY = "repos";
+
 	private static Map<String,String> repoNames;
+	private String repoUserDefault;
+	private String repoPwdDefault;
+	private String repoBaseUrl;
 
-	public RemoteRepo(Project project) {
+	// TODO JHE: Not sure we want to throw an exception here ... but if the JSON isn't found , well, we can't really go further...
```
Comment (chhex) [inline common-repo/src/main/java/com/apgsga/gradle/repo/extensions/RemoteRepo.java]: A Runtime Exception with a meaningful message will.... these checked Exception in Java are tedious anyway, to say the least

Comment (apgsga-jhe) [reply common-repo/src/main/java/com/apgsga/gradle/repo/extensions/RemoteRepo.java]: @chhex : I just commited a change in order to throw a RuntimeException, was that more or less what you meant?

**Comment by apgsga-jhe** [comment]
@chhex : Ok, last "throws" has been removed. Regard test classes, "--init" and "--stacktrae" moved to abstract class, i'm merging this one.
@apgsga-stb : FYI

**Pull request merged**